config/static.c

```diff
@@ -26,103 +35,121 @@
 #include "gatekeeper_lls.h"
 #include "gatekeeper_rt.h"
 
+#define RTE_LOGTYPE_CONFIG RTE_LOGTYPE_USER2
+
+/* TODO Get the install-path via Makefile. */
+#define LUA_BASE_DIR "./lua/"
+#define LUA_CONFIG_FILE     "gatekeeper_config.lua"
+
+struct gatekeeper_config {
+	struct bp_config 	*bp_conf;
+	struct catcher_config 	*catcher_conf;
+	struct dynamic_config 	*dy_conf;
+	struct cps_config 	*cps_conf;
+	struct ggu_config 	*ggu_conf;
+	struct gk_config 	*gk_conf;
+	struct gt_config 	*gt_conf;
+	struct rt_config 	*rt_conf;
+};
+
+static lua_State *lua_state;
```

Comment:
I am considering how to simplify the code. Let's say, if Gatekeeper needs to exit by receiving monitored signal, I think we should destroy the states. If so, where should we put these states destroying functions? Considering each functional block may run multiple instances. For states belongs to an instance,  they can be destroyed by the instances themselves; for states shared by multiple instances of a block, or states shared by multiple functional blocks (e.g., the mailboxes), how do we destroy them?